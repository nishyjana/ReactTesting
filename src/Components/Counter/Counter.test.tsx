import { render } from "@testing-library/react";
import React from "react";

describe("This is the browser testing", () => {
  //snapshot testing for the counter
  test("Snapshot test for the counter", () => {
    const { asFragment } = render(<button>Click</button>);
    expect(asFragment()).toMatchSnapshot();
  });
});
