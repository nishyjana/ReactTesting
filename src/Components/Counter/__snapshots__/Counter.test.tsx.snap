// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is the browser testing Snapshot test for the counter 1`] = `
<DocumentFragment>
  <button>
    Click
  </button>
</DocumentFragment>
`;
